src/devtools/devtools-popup.component.tsx

```diff
@@ -10,11 +11,15 @@ export default function DevToolsPopup(props: DevToolsPopupProps) {
       <Tabs className={styles.tabs}>
         <TabList className={styles.tabList}>
           <Tab>Import Map</Tab>
+          <Tab>Backend Modules</Tab>
         </TabList>
         <TabPanels className={styles.tabPanels}>
           <TabPanel>
             <ImportMap toggleOverridden={props.toggleOverridden} />
           </TabPanel>
+          <TabPanel>
+            <BackendModule></BackendModule>
```

Comment:
Let's make the Devtools rectangle trigger button thing change colors when there is a backend module dependency mismatch.

This can be accomplished with `toggleOverridden`. See how the ImportMap component is calling that function to see an example of how.

One thing is that we'll need to account for whether zero, one, or both of the tabs in the devtools are causing the trigger to change colors.